oasislmf/utils/defaults.py

```diff
@@ -59,6 +64,47 @@
     if cov_type in ['buildings', 'other', 'contents', 'bi']
 })
 
+SOURCE_FILENAMES = OrderedDict({
+    'loc': 'source_location.csv',
+    'acc': 'source_account.csv',
+    'info': 'source_info.csv',
+    'scope': 'source_scope.csv'
+})
+
+# Store index from merged source files (for later slice & dice)
+SOURCE_IDX = OrderedDict({
+    'loc': 'loc_idx',
+    'acc': 'acc_idx',
+    'info': 'info_idx',
+    'scope': 'scope_idx'
+})
+
+
+SUMMARY_MAPPING = OrderedDict({
+    'gul_map_fn': 'gul_summary_map.csv',
+    'fm_map_fn': 'fm_summary_map.csv',
+    #'ri_map_fn': 'summary_map.csv',
+})
+
+SUMMARY_OUTPUT = OrderedDict({
+        'gul': 'gulsummaryxref.csv',
+        'il': 'fmsummaryxref.csv',
+})
+
+
+## Update with load OED column names 
+SUMMARY_GROUPING = OrderedDict({
```

Comment:
As I mentioned in the Slack PM, it would be better to create this dict by loading it from a static JSON profile, say a summary profile, that you store in the package static data. You can create a wrapper method to load it in `utils/defaults.py`. Column names of source files, or any intermediate files, should not be hard coded - this just introduces the possibility of runtime errors.